pyproject.toml

```diff
@@ -5,9 +5,6 @@ requires = [
 ]
 build-backend = "setuptools.build_meta"
 
-[tool.nbqa.mutate]
-black = 1
```

Comment:
I was assuming you wanted auto formatting because I saw this construct, which is deprecated, so I replaced it with the alternative. (I was wondering why there were quite a few notebook changes, but most of them seems to be empty space at the end of lines)

Reply:
This is a left-over of that PR that I haven't cleaned up.